pkg/internalapis/cnsoperator/apis.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apis
```

Comment:
What is the golang/CRD standard for directory structure that involves internal APIs? 

Can we have "pkg/internal/api" instead of "pkg/internalapis/"? This depends on the answer to the first question.

Reply:
There is no standard structure from golang.